### PR TITLE
K.Zhang_hw01

### PR DESCRIPTION
**Pull request by klyzhng**
Name: Kelly Zhang
What I did: I added information about my year, where I've lived, my hobbies and interests, my involvement in campus organizations and departments, and my reason for taking this class. I also inserted a meme of a seal.
How I did it: I just typed in the README and used the notation given in the R Markdown guide to include some formatting variety.
Reflection & Problems: I thought the work was doable. I'm still getting the hang of making commits and pushing them to Git Hub; I ran into a problem when I tried to override a specific previous commit as I had noticed that the resulting text wasn't formatted as I had intended. I noticed that I couldn't push a new commit that would fix a past piece of my code that had already been committed. I was able to edit the problematic commit through Git Hub but later after typing some more in RStudio, I was unable to push new commits because the online and local versions of the README were not "integrated" (according to the error message). I then tried codes in the shell that I found on StackExchange and other online forums but wasn't able to fix this problem. I was going to post this issue in the forum but it was taking me too long to write about it and I didn't understand much of the codes discussed online, so I restarted the process. I saved my biography in a Google Doc, deleted both projects and hw01 folders (online and local), and forked the hw01 repository again, ..., pasted my text and made commits then pushed them to Git Hub. I'd appreciate advice on how to prevent this restarting process and directly edit past commits. The other problem I encountered was not knowing how comprehensive my biography was supposed to be, so I just assumed I was presenting myself to a fellow peer/friend.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### A Glimpse of My Life
+Welcome to my blurb! I'm Kelly Zhang and a rising second-year in the College. I'll be telling you a bit more about myself today.
+
+***Places of Residence*** I was born in St. Paul, MN and moved to Santa Fe, NM and later to the suburbs of Washington D.C., where I still reside today.
```
Comment (YinsuH) [inline README.md:6]: The use of bolded and italic words makes the doc really easy to read!

**Review by YinsuH — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |      x     |              |            |

Remarks:
Your presentation and writing are really creative! The use of headings makes the structure of the doc really easy to follow. The code runs without any problems as well!

File: README.md (hunk 1 above)
Comment (nearridge) [inline README.md:6]: Nice use of the triple `*` to italics and bold!

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+***Places of Residence*** I was born in St. Paul, MN and moved to Santa Fe, NM and later to the suburbs of Washington D.C., where I still reside today. 
+
+***Hobbies and Interests*** I've tried a handful of sports ranging from gymnastics to figure skating to field hockey but my favorite is skating. I skated on a synchronized skating team for three years and later played varsity field hockey for two years. 
+
+Outside of sports, I enjoy...
+* amateur gardening-my pea sprouts unfortunately withered away a few days ago after weeks of loving care
+* browsing space and airplane related news and memes
+* experimenting with online applications to track flights
+* zooming with friends
+* wasting away on Netflix
```
Comment (nearridge) [inline README.md:15]: Great list here.

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
...
+
+***UChicago Involvement*** I'm also a member of Salisbury's House Council, The Blue Chips, Army ROTC, and Student Alumni Committee. I am also a prospective math/CAMS, statistics, or economics (surprise!) major.
+
+***Near-Future Vision*** I enrolled in this course in part because of the requirements of UChicago's Summer Institute in Social Research Methods program and also because I had a taste of R this past quarter in a statistics class and wanted to build a stronger foundation in the language. I look forward to getting to know you all through this course and I can't wait to feel the pure happiness this seal is experiencing: <br />
+![Image of Seal](https://www.probytes.net/wp-content/uploads/2018/01/1.jpg)
```
Comment (nearridge) [inline README.md:20]: Nice job embedding this image.

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |         x     |            |
| **Reproducibility**         |         x  |              |            |

Remarks: Great job with this assignment. No major issues here.

**Pull request closed without merging**